docs/sql-ref-syntax-ddl-alter-table.md

```diff
@@ -225,6 +294,25 @@ ALTER TABLE table_identifier [ partition_spec ] SET LOCATION 'new_location'
 
     Specifies the SERDE properties to be set.
 
+### RECOVER PARTITIONS
+
+`RECOVER PARTITIONS` statement recovers all the partitions in the directory of a table and updates the Hive metastore.
```

Comment:
nit: maybe 

```sql
`ALTER TABLE RECOVER PARTITIONS` statement blah blah...
```

to keep the consistency with other items ?

Reply:
thanks for the review. fixed.